platform/src/main/java/org/stellar/anchor/platform/event/EventProcessorManager.java

```diff
@@ -93,11 +93,13 @@ public void start() {
         switch (clientConfig.getType()) {
           case CUSTODIAL:
             processorName =
-                CLIENT_STATUS_CALLBACK_EVENT_PROCESSOR_NAME_PREFIX + clientConfig.getSigningKey();
+                CLIENT_STATUS_CALLBACK_EVENT_PROCESSOR_NAME_PREFIX
+                    + clientConfig.getSigningKeys().stream().findFirst();
```

Comment:
If there are multiple signing keys, will this break? Should we use string concatenation instead of the first key?

Reply:
This won't break, it will just find the first element. 
But it could returns any element from the Stream since SigningKeys and domains are sets (i.e no encounter order), we'd better just stream all the values